git.go

```diff
@@ -258,7 +257,6 @@ func (r *Repo) fetch(h Hash) (fs.FS, error) {
 		if len(line) == 0 || line[0] == 0 || line[0] > 3 {
 			fmt.Printf("%q\n", line)
 			continue
-			return nil, fmt.Errorf("fetch: malformed response: invalid sideband: %q", line)
```

Comment:
Note that it would be better to delete the 2 lines above this line, and keep this line. Compare with golang/go#70688 and https://github.com/golang/website/commit/84bc8a71547811f19ea901d0c8e26f527c4328c7.